Resources/doc/filters/general.rst

```diff
@@ -0,0 +1,204 @@
+
+.. default-role:: code
+
+General Filters
+===============
+
+Background
+----------
+
+.. _filter-background:
+
+The built-in `background` filter performs layer transformations
+(which includes creating and mergin layer operations). This
+filter exposes a number of `background options`_ which may be used
+to configure its behavior.
+
+Example configuration::
```

Comment:
This line makes the following code be treated like PHP instead of YAML.

Replace this:

```
Example configuration::
```

by this:

```
Example configuration:

.. code-block:: yaml
```